changelog.d/4643.misc

```diff
@@ -0,0 +1 @@
+Replacing search icon by filter icon
```

Comment:
might be worth mentioning where the icon is replaced

Reply:
Yes indeed! I have added more details.